packages/react/src/components/Skeleton/Rectangle/Rectangle.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import { useSynchronizedAnimation } from '../../../hooks';
+import classes from '../Skeleton.module.css';
+
+export type RectangleProps = {
+  width?: string | number;
+  height?: string | number;
+} & HTMLAttributes<HTMLDivElement>;
+
+/**  Skeleton component used for indicating loading elements of circular shape */
+export const Rectangle = ({
+  width,
+  height,
+  className,
+  children,
+  ...rest
+}: RectangleProps) => {
+  const ref = useSynchronizedAnimation('opacity-fade');
```

Comment:
We would need to get the animation name from the component, since it will change in build.
```js
let animations = document.querySelector(".animate").getAnimations();
console.log(animations[0].animationName);
```

You would need to change the code to work with react, but we should be able to get the `animationName` with this

Reply:
I hadn't even thought about this, but you're right of course - we're lucky to have you here! 😄

I couldn't get this line of code to work though, I just got that animations was undefined.
However, I found out that we can actually just get the modular animation name the same way we do with classes - by just referencing them through the imported css-file:
`console.log(classes['opacity-fade']);`
![image](https://github.com/digdir/designsystem/assets/22012997/3aa366a9-be3e-418f-8b11-67fc99488382)

I would think that would be enough, and that it's so simple we won't need a hook for it, but what do you think?

Reply:
Awesome find, try building and packing it and linking the package the package locally to see - but looks good!